english/permissions.py

```diff
@@ -0,0 +1,28 @@
+from rest_framework.permissions import BasePermission
+from users.models import User
+
+
+class IsEmployee(BasePermission):
+    def has_permission(self, request, view):
+        return bool(
+            request.user and
+            request.user.is_authenticated and
+            request.user.role == User.EMPLOYEE
+        )
+
+    def has_object_permission(self, request, view, obj):
+        is_employee = self.has_permission(request, view)
```

Comment:
Do you need this?

Reply:
I am using it not to duplicate code because the function has_object_permissons has similar validators as function has_permission. But the function has_object_permissons has one more validator as 'obj.user == request.user'.
I can remove `is_employee = self.has_permission(request, view)` but it will look's not so good

Reply:
Please, check how many times calls has_permission function for one request